lib/middleware/json_parser.js

```diff
@@ -0,0 +1,34 @@
+var _ = require('lodash');
+
+var bodyParser = require('body-parser').json();
+
+var getError = require('../error_helper').getError;
+
+var BAD_REQUEST = 400;
+
+var BAD_REQUEST_ERR = {
+  developerMessage: 'The JSON message sent is invalid',
+  userMessage: 'Something went wrong. Please review your data.',
+  code: 400
+};
+
+function translateError(err) {
+  var isInvalidJSONErr = err instanceof SyntaxError &&
+    _.get(err, 'status') === BAD_REQUEST;
+
+  if (isInvalidJSONErr) {
+    return getError({}, BAD_REQUEST_ERR);
+  }
+  return err;
```

Comment:
If err is present should always return a 400, shouldnt expose the actual error or i am missing something else.